src/coreos.rs

```diff
@@ -33,7 +32,21 @@ const ALEPH_PATH: &str = "/sysroot/.coreos-aleph-version.json";
 
 pub(crate) fn get_aleph_version() -> Result<Option<AlephWithTimestamp>> {
     let sysroot = openat::Dir::open("/")?;
```

Comment:
Oh actually since we always operate on `/` here we can just use the `std` APIs too.  I can push a change to do that if you like.

Reply:
Done